### PR TITLE
Use Breadcrumbs from @backstage/core

### PR DESCRIPTION
**Pull request by wrousseau**
## Hey, I just made a Pull Request!

Those are the remaining change to remove all usage of material-ui's Breadcrumbs to close issue #4641. Also refactored to use `Box` as suggested in the previous pull request 

Circle-ci plugin (I've tweaked the code so it shows, don't mind the content of the table)
Capture d’écran 2021-02-25 à 20 07 31

Cloudbuild plugin (same)
Capture d’écran 2021-02-25 à 20 10 26

Github actions
Capture d’écran 2021-02-25 à 20 20 44

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4705-un6wmagwcshdvwfeloblrqieoa2clbqz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6037f8e465f8172a53fab11c

File: plugins/cloudbuild/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx
Comment (adamdmharvey) [inline plugins/cloudbuild/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx:92]: I know it was here before, but I'm interested that this uses the same terminology as GitHub Actions. I wonder if this was a copy and paste to get this plugin out there.

https://cloud.google.com/build/docs/overview

It seems like GCP calls these Builds and Build Results but I could be wrong? do we know anyone who knows more about it? Spotify is a big GCP shop, any ideas?

**Review by adamdmharvey — COMMENTED** [review]
woo! banging them all out in one fell swoop :)

File: plugins/cloudbuild/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx
Comment (wrousseau) [reply plugins/cloudbuild/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx:92]: Capture d’écran 2021-02-25 à 20 37 08
Capture d’écran 2021-02-25 à 20 37 13

I suggest using "Build history" and "Build details" as suggested in the screenshots I took from a video demo. I can easily include the change in this PR if you're ok with it

Comment (adamdmharvey) [reply plugins/cloudbuild/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx:92]: @ebarriosjr you may have been one of the early developers on this one, might you know?

Comment (adamdmharvey) [reply plugins/cloudbuild/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx:92]: Oh awesome yeah @wrousseau nice one! Yeah I just figure you're in there, it's a small change, may as well avoid another PR loop. May be best to add a second changeset just for CloudBuild and say the terminology has updated (so leave the one you have, and add one more) but if you don't mind squeezing it in I think a good incremental alignment!!!

Comment (wrousseau) [reply plugins/cloudbuild/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx]: @adamdmharvey done :)

**Review by adamdmharvey — APPROVED** [review]
looks great!!! :tada: just need core team signoff and we're off..... thanks so much!

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4705-un6wmagwcshdvwfeloblrqieoa2clbqz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6037f8e465f8172a53fab11c

**Review by Rugvip — APPROVED** [review]
Yep, looks good, thank you! 👍 

And thank you too @adamdmharvey ❤️

**Pull request merged**
